config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
I just downloaded elasticsearch 5.0 alpha5, and format is still custom `yaml` format and log4j 1.x is still used (it's like that since elasticsearch 1.x).  
I checked : no branch or PR concerning migration to log4j 2.x is existing.  

Concerning "dumb config as a starter", in log4j 1.x, properties format was really easy for a start.  
But in log4j 2.x, things have changed : the properties format is now more complex and verbose to let the same hierarchical capacities than xml format.
I especially noticed that you have copied/pasted the only log4j2 properties example as it is :)